lib/cdo/mysql_console_helper.rb

```diff
@@ -11,6 +11,14 @@ def self.run(connection_uri, args)
     command << "--execute=\"#{args}\"" unless args.empty?
     command << "--password=#{db.password}" unless db.password.nil?
 
+    warning =
+      "****************************************************************\n"\
+      "*** You are connecting to the master production database.    ***\n"\
+      "*** For simple queries, please use the reporting db instead. ***\n"\
```

Comment:
nit: The issue isn't so much query complexity, but rather whether the query needs to run against the master. Perhaps:
```
You are connecting to the master production database.
Please connect to the reporting database instead via
bin/dashboard-reporting-sql or bin/pegasus-reporting-sql.
```